layouts/default-max.vue

```diff
@@ -18,6 +19,8 @@ export default class MaxLayout extends Vue {
   onClick (e: CustomEvent) {
     this.$trackClickEvent(`${e.detail?.label?.toLowerCase()}`, 'menu')
   }
+
+  isLearnPage: boolean = this.$route.path === '/learn/course'
```

Comment:
@mariacloehb for this, I think we need to consider both scenarios:

if the path _is equal to_ `/learn` OR if the path _includes_ `/learn/course`

Reply:
it should be 
```suggestion
  isLearnPage: boolean = this.$route.path.includes('/learn')
```

why?:
- `this.$route.path === '/learn/course'` will only recognized the URL `qiskit.org/learn/course`, so it won't work with `qiskit.org/learn` or `qiskit.org/learn/course/<course-name`
- `this.$route.path.includes('/learn/course')` will recognize `qiskit.org/learn/course/<course-name`, but not the URL `qiskit.org/learn`

Reply:
great point... I like this approach better